dev_guide/templates.adoc

```diff
@@ -82,13 +121,27 @@ $ oc create -f <filename> -n <project>
 The template is now available to be selected for a configuration using the web
 console or the CLI.
 
-=== Generating a Configuration
-Generate a configuration with the following command:
+=== Generating a list of resources
+Generate a list of resources with the following command:
 
 ----
 $ oc process -f <filename>
 ----
 
+You can also process the uploaded template by calling the process command using:
```

Comment:
?? generate is for generating values... processing is for template, I don't see your point ;-)